app/src/main/res/layout/activity_presentation1.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:animationCache="false"
+    tools:context=".PresentationActivity1">
+
+    <TextView
+        android:id="@+id/pres1_textView1"
```

Comment:
Maybe a more relevant id instead of textView1 / textView2...

Reply:
Well, they are prepended by pres1 that stands for presentation 1.
But I was going to put more explicit names once we really populate that area.